### DISCUSSION TITLE
environmental variables

### DISCUSSION THREAD

**Discussion by RRConstruct** [serverless/serverless#10193] [Q&A]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest version?

- [X] Yes, I'm using the latest version

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

Want to know how the object environmental can be used / configured?
for example: 

const example= {
TEST1 : process.env.TEST1_ID,
TEST2 : process.env.TEST2_ID,
TEST3 : process.env.TEST3_ID,
}
thanks in advance and looking forward for the solution 

### Service configuration (serverless.yml) content

```yaml
N/A
```

### Command name and used flags

N?A

### Command output

```shell
N/A
```

### Environment information

```shell
"serverless-bundle": "^1.9.1"
```

**Comment by pgrzesik** [comment]
This is not a bug but rather a request for help with specific use case - I'll move it to discussions 

As for the problem itself - could you clarify how you want to use environment variables? As a part of Lambda code or serverless configuration?
